python/paddle/fluid/tests/unittests/test_dist_mnist_dgc_nccl.py

```diff
@@ -76,10 +78,12 @@ def test_dist_train(self):
                 log_name=flag_name)
 
     def tearDown(self):
-        result = count_of_sparse_all_reduce_calls(
-            'test_dist_mnist_dgc_nccl_dgc_2cards_local.log')
-        # same as above, but use two cards
-        self.assertEqual(result, 6)
+        import paddle.fluid as fluid
```

Comment:
Why import twice?

Reply:
two tests, one test is two processes, one test is one process two cards